contracts/MembersLibrary.sol

```diff
@@ -50,7 +50,7 @@ library MembersLibrary {
         // mapping subset memberIndex to superset index
         memberStore.setIntIndex(sha3('memberIndex'), memberIndex, index);
         memberStore.setAddressValue(sha3('account', index), account);
-        // memberStore.setAddressIndex('account', account, index);
+        memberStore.setAddressIndex(sha3('account'), account, index);
```

Comment:
As proposed by you, updating sha3('account') to sha3('accountIndex'). We can also change the variable name index to accountIndex for more clarity.